sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/IntervalBenchmark.scala

```diff
@@ -85,7 +85,8 @@ object IntervalBenchmark extends SqlBasedBenchmark {
     val timeUnits = Seq(
       "13 months", "                      1                     months",
       "100 weeks", "9 days", "12 hours", "-                    3 hours",
-      "5 minutes", "45 seconds", "123 milliseconds", "567 microseconds")
+      "5 minutes", "45.123456 seconds", "123 milliseconds", "567 microseconds",
+      "98.76543210 seconds", "12.34567890 seconds", "99.999999999 seconds")
```

Comment:
Only add `seconds` decimals for the master branch to support, so we can compare it with our PR in a fair play. Also, parsing other units should be as same as the seconds.